sirepo/feature_config.py

```diff
@@ -115,7 +116,7 @@ def b(msg, dev=False):
     _cfg = pkconfig.init(
         # No secrets should be stored here (see sirepo.job.agent_env)
         api_modules=((), set, 'optional api modules, e.g. status'),
-        default_proprietary_sim_types=(set(), set, 'codes where all users are authorized by default but that authorization can be revoked'),
+        proprietary_with_moderation_prompt_sim_types=(set(), set, 'codes where all users are authorized by default but that authorization can be revoked'),
```

Comment:
The documentation string is incorrect: codes where all users must be authorized via moderation.

Reply:
84a51a4c4d292b403b922c73411c9bad58de76bf